### PR TITLE
[ACCEL] When applied, this commit will resolve conflict for tests coverage

### PR DESCRIPTION
**Pull request by Golub-Sergey**
**Description:**
Fix edx-ora2 errors and failures, make some setup of email_notification for correct test work, add try-catch in submission_mixin in xblock

**Youtrack:**
[ticket](https://youtrack.raccoongang.com/issue/ACCEL-396), [review ticket](https://youtrack.raccoongang.com/issue/ACCEL-402)

### CODE REVIEW

File: openassessment/xblock/test/test_staff_area.py [hunk 1]
```diff
@@ -590,22 +574,40 @@ def test_cancel_submission_full_flow(self, xblock):
...
         # Create a submission for Bob, and corresponding workflow.
         submission = self._create_submission(bob_item, {'text': "Bob Answer"}, ['peer'])
-
         incorrect_submission_uuid = 'abc'
         params = {"submission_uuid": incorrect_submission_uuid, "comments": "Inappropriate language."}
         # Raise flow not found exception.
-        resp = self.request(xblock, 'cancel_submission', json.dumps(params), response_format='json')
-        self.assertIn("Error finding workflow", resp['msg'])
-        self.assertEqual(False, resp['success'])
+        with self.assertRaisesMessage(SubmissionInternalError, '[u"\'abc\' is not a valid UUID."]'):
```
Comment (flying-pi) [inline openassessment/xblock/test/test_staff_area.py]: Is it correct, that we check, that error has happened? Maybe we should investigate why this exception raised?

File: openassessment/xblock/test/test_submission.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
 from submissions.api import SubmissionInternalError, SubmissionRequestError
 
 from .base import XBlockHandlerTestCase, scenario
+from mock import patch
```
Comment (flying-pi) [inline openassessment/xblock/test/test_submission.py]: Could you group imports, which relates to the mock package?

File: openassessment/xblock/test/test_staff_area.py (hunk 1 above)
Comment (Golub-Sergey) [reply openassessment/xblock/test/test_staff_area.py]: Done

File: openassessment/xblock/test/test_submission.py (hunk 2 above)
Comment (Golub-Sergey) [reply openassessment/xblock/test/test_submission.py]: Done

File: openassessment/xblock/test/test_staff_area.py [hunk 3]
```diff
@@ -590,22 +573,40 @@ def test_cancel_submission_full_flow(self, xblock):
...
         # Create a submission for Bob, and corresponding workflow.
         submission = self._create_submission(bob_item, {'text': "Bob Answer"}, ['peer'])
-
         incorrect_submission_uuid = 'abc'
         params = {"submission_uuid": incorrect_submission_uuid, "comments": "Inappropriate language."}
         # Raise flow not found exception.
-        resp = self.request(xblock, 'cancel_submission', json.dumps(params), response_format='json')
-        self.assertIn("Error finding workflow", resp['msg'])
-        self.assertEqual(False, resp['success'])
+        with patch('openassessment.xblock.staff_area_mixin.submission_api.get_submission_and_student') as mock_get_submission:
```
Comment (flying-pi) [inline openassessment/xblock/test/test_staff_area.py]: Could we use patch here as a decorator?

Comment (Golub-Sergey) [reply openassessment/xblock/test/test_staff_area.py]: Done

**Pull request merged**